gax/package.json

```diff
@@ -13,18 +13,20 @@
     "@grpc/grpc-js": "~1.9.6",
     "@grpc/proto-loader": "^0.7.0",
     "@types/long": "^4.0.0",
+    "@types/uuid": "^9.0.7",
```

Comment:
This should go to `devDependencies` unless there is a reason not to.  `@types/long` is an exception, it's a dependency of the generated `.d.ts` files so we must list it as if it is a runtime dependency. Other types go to devDeps.